doc/changes/latest.inc

```diff
@@ -24,6 +24,7 @@ Current (1.5.dev0)
 Enhancements
 ~~~~~~~~~~~~
 - Add ``cmap`` argument for the :func:`mne.viz.plot_sensors` (:gh:`11720` by :newcontrib:`Gennadiy Belonosov`)
+- Return unmodified instance if new sampling frequency is identical to the original in :meth:`mne.io.Raw.resample`, :meth:`mne.Epochs.resample`, :meth:`mne.Evoked.resample` and :meth:`mne.SourceEstimate.resample` (:gh:`11736` by :newcontrib:`Gennadiy Belonosov`)
```

Comment:
You'll need to add your name to `doc/changes/names.inc` as well

Reply:
Yeah, I've already added myself there on the previous PR.